packages/config/config/devices/0x0384/ha-zw-5sab.json

```diff
@@ -187,29 +182,17 @@
 		},
 		{
 			"#": "13",
-			"label": "Pir Triggers The Waiting Time Value",
-			"description": "Pir triggers the waiting time value",
+			"label": "Pir Triggers Waiting Time Value",
+			"description": "Range: 0x05-0x3BC4",
 			"valueSize": 2,
 			"minValue": 0,
 			"maxValue": 30,
 			"defaultValue": 30,
```

Comment:
This does not match up. Can you share the manual for this parameter?

Reply:
[AIBASE Z-Wave Multi-Sensor UserGuide 20180606.pdf](https://github.com/zwave-js/node-zwave-js/files/8950326/AIBASE.Z-Wave.Multi-Sensor.UserGuide.20180606.pdf)
![param-13](https://user-images.githubusercontent.com/44245179/174840269-071b9105-2bbf-4057-b3fb-956dae9130c6.png)

Reply:
ok the min/max value was just wrong then. I'm going to assume these are seconds